modules/local/cellprofiler/illuminationcorrection/main.nf

```diff
@@ -0,0 +1,55 @@
+process CELLPROFILER_ILLUMINATIONCORRECTION {
+    tag "${meta.id}"
+    label 'process_single'
+
+    // TODO nf-core: List required Conda package(s).
+    //               Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    //               For Conda, the build (i.e. "h9402c20_2") must be EXCLUDED to support installation on different operating systems.
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
+    conda "${moduleDir}/environment.yml"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://depot.galaxyproject.org/singularity/cellprofiler:4.2.8--pyhdfd78af_0'
+        : 'community.wave.seqera.io/library/cellprofiler:4.2.8--aff0a99749304a7f'}"
+
+    input:
+    tuple val(meta), path(load_data_csv)
+    path image_dir, stageAs: 'images'
+    path illumination_pipeline
+
+    output:
+    tuple val(meta), path("illum_corrected_images/*.npy"), emit: illum_corrected_images
```

Comment:
```suggestion
    tuple val(meta), path("illum_corrected_images/*.npy"), emit: illumination_corrections
```
The npy files are corrections not actual images themselves.

Reply:
oh right! fixed